restalchemy/storage/sql/dialect/mysql.py

```diff
@@ -24,6 +24,13 @@
 from restalchemy.storage.sql.dialect import exceptions as exc
 
 
+def get_savepoint_expressions(name):
```

Comment:
savepoints may be hotpaths, maybe use some dict with mapping? Point is - function will be evaluated each time. Looks pretty static for me.